images/api/src/Auth.js

```diff
@@ -47,15 +52,31 @@ class Auth {
                   organisation: data[0].organisation
                 })
               } else {
-                res.send(401, {message: "No user found"});
+                if(req.params.path.includes("admin")) {
```

Comment:
Include the "/" before and after "admin" in this check please, just for safety measures